datafusion/core/src/datasource/physical_plan/parquet/row_groups.rs

```diff
@@ -169,8 +174,8 @@ pub(crate) async fn prune_row_groups_by_bloom_filters<
 
 /// Implements `PruningStatistics` for Parquet Split Block Bloom Filters (SBBF)
 struct BloomFilterStatistics {
-    /// Maps column name to the parquet bloom filter
-    column_sbbf: HashMap<String, Sbbf>,
+    /// Maps column name to the parquet bloom filter and parquet physical type
+    column_sbbf: HashMap<String, (Sbbf, Type)>,
```

Comment:
because the sbbf `check` input is parquet type, so we need know the read `parquet physical type` when construct check value 🤔 , is there any another way